css/style.css

```diff
@@ -1,4 +1,543 @@
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+}
+
+.test__column:nth-child(3) {
+  margin: 10px 0 0 0;
+}
+
+.test__item {
+  padding: 0 7px 0 0;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+}
+
+.test__right {
+  padding: 0;
+}
+
+.test__column_testdrive {
+  padding: 10px 0 0;
+}
+
+.test__body {
+  font-weight: 600;
+  font-size: 35px;
+  margin: 0;
+}
+
+.test__body_small {
+  font-size: 20px;
+}
+
+.test__body_bottom {
+  font-size: 12px;
+  line-height: 15px;
+  margin: 1px 0 0;
+}
+
+.test__body_testdrive {
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 24px;
+  text-align: center;
+  text-transform: uppercase;
+  padding: 0 0 0 30px;
+  color: #000;
+  text-shadow: 3px 5px 10px rgba(0, 0, 0, .33);
+  background: url(../img/icons/steering-wheel.png) no-repeat;
+}
+
+.test__body_testdrive:hover {
+  color: #e51937;
+  -webkit-transition: ease-in-out .5s;
+  -o-transition: ease-in-out .5s;
+  transition: ease-in-out .5s;
+  background: url(../img/icons/steering-wheel_red.png) no-repeat;
+}
+
+.btn {
+  width: 160px;
+  height: 40px;
+  border: 2px solid #e51937;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #e51937;
+  padding: 10px 0 0 0;
+}
+
+.btn__test {
+  color: #e51937;
+  border: 2px solid #e51937;
+}
+
+.btn__test:hover {
+  background-color: #e51937;
+  color: #fff;
+  -webkit-transition: ease-in-out .5s;
+  -o-transition: ease-in-out .5s;
+  transition: ease-in-out .5s;
+}
+
+.button {
+  margin: 5px 0 0;
+}
+
+.test__img {
+  display: block;
+  margin: 5px  auto 0;
+  width: 100%;
+  height: 100%;
+  -ms-flex-item-align: start;
+  align-self: flex-start;
+}
+
+.test__arrow {
+  margin: 39px auto 0;
+  width: 30px;
+}
+
+@media (max-width: 768px) {
+  .header__right_row {
+    display: -webkit-flex;
+    display: -moz-flex;
+    display: -ms-flex;
+    display: -o-flex;
+    display: flex;
+    width: 35px;
+    justify-content: flex-end;
+  }
+
+  .menu__item,
+  .menu__row .menu__item:first-child {
+    display: none;
+  }
+
+  .test {
+    margin: 40px 0 0;
+  }
+
+  .test__row {
+    margin: 0 auto;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    max-width: 490px;
+    padding: 0;
+  }
+
+  .test__row_right {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 100%;
+    flex: 0 1 100%;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    max-width: 488px;
+    padding: 0;
+    margin: 0 auto;
+  }
+
+  .test__row_left {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 418px;
+    flex: 0 1 418px;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    padding: 0;
+    margin: 55px auto 0;
+    max-width: 418px;
+  }
+
+  .test__item_left{
+  justify-content: flex-end;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5804302)